starter-code/css/main.css

```diff
@@ -6,3 +6,189 @@ Green: #00B172
 White: #FFF
 
 */
+
+* {
+  /* border: 1px solid red; */
+  padding: 0px;
+  margin: 0px;
+}
+
+body {
+  font-family: Arial, Helvetica, sans-serif
+}
+
+.wrapper {
+  width: 900px;
+  margin: 0 auto;
+}
+
+.header {
+  height: 60px;
+  line-height: 40px;
+}
+
+.header .logo {
+  height: 40px;
+  float: left;
+  padding: 10px 15px;
+}
+
+.logo-jump:hover {
+  animation: bounce 1.2s ease-out;
+}
+
+.header .nav {
+  float: right;
+  padding: 10px 15px;
+}
+
+.header .nav ul {
+  list-style-type: none;
+}
+
+.header .nav li {
+  display: inline;
+  padding: 0 5px;
+  color: grey;
+  font-size: .9em;
+  font-weight: lighter;
+}
+
+.banner {
+  width: 900px;
+  background: url("../images/landing.jpg") no-repeat center center;
+  background-size: 900px;
+  height: 80vh;
+  text-align: center;
+}
+
+.banner h1, .banner p {
+  color: white; 
+  position: absolute;
+  top: 35%; left: 50%;
+  transform: translate(-50%,-50%);
+}
+
+.banner h1 {
+  font-size: 3em;
+  font-weight: 400;
+  top: 35%; left: 50%;
+  transform: translate(-50%,-50%);
+}
+
+.banner p {
+  font-size: 1em;
+  font-weight: 200;
+  line-height: 1.5em;
+  top: 50%; left: 50%;
+  transform: translate(-50%,-50%);
+}
+
+.feature-section {
```

Comment:
If you want to check what I did, here is my [Spotify code](https://github.com/VivianSolide/lab-css-spotify-clone/blob/master/starter-code/css/main.css)
For the green trait you could try this:
```css
.wos > h3 {
  margin-top: 10vh;
  text-align: center;
  font-size: 3rem;
  border-bottom: 0.5rem solid #00b172;
  padding-bottom: 10px;
  letter-spacing: 1px;
  display: inline-block;
}
```
Otherwise, the pseudo class after seen yesterday is aswell a very good way to do this.